cloverage/test/cloverage/coverage_test.clj

```diff
@@ -361,9 +362,19 @@
             "cloverage.sample.exercise-instrumentation")
            0))))
 
-(defn- equal-content? [fname dir-a dir-b]
-  (= (slurp (io/file dir-a fname))
-     (slurp (io/file dir-b fname))))
+(defn remove-dynamic-strings
+  "Removes strings that are dynamic/unpredictable, for a stable comparison"
+  [s]
+  (-> s
+      (str/replace #"\"service_job_id\":\"\d+\""
+                   "\"service_job_id\":null")
+      (str/replace "\"service_name\":\"circleci\""
+                   "\"service_name\":null")))
+
+(defn- assert-equal-content! [fname dir-a dir-b]
+  (t/is (= (remove-dynamic-strings (slurp (io/file dir-a fname)))
```

Comment:
By moving the `is` here, one gets Kaocha diffing which helped me a lot debug the issue!